payout.go

```diff
@@ -2,17 +2,16 @@ package paypalsdk
 
 import "fmt"
 
-// CreateSinglePayout submits a payout with a synchronous API call, which immediately returns the results of a PayPal payment.
+// CreateSinglePayout submits a payout with an asynchronous API call, which immediately returns the results of a PayPal payment.
```

Comment:
So does this mean async mode is the only option moving forward?  Do you need to explicitly specific `sync_mode=false` as per the error message you reported in the issue?  Should probably update this comment to not say "which immediately returns ..."

Reply:
according to the [paypal docs for payouts](https://developer.paypal.com/docs/api/payments.payouts-batch#payouts) , by default a payout is asynchronous, so I don't need to explicitely include `sync_mode=false`

```
By default, a Payouts API call is asynchronous, which lets you show payout details at a later time. 
```

You're right in that I need to update the comments, since the payout results aren't immediately returned.  I'm removing that part of the comment